sn_node_manager/src/local.rs

```diff
@@ -286,8 +296,31 @@ pub async fn run_node(
     launcher: &dyn Launcher,
     rpc_client: &dyn RpcActions,
 ) -> Result<NodeServiceData> {
+    let user = match get_username() {
```

Comment:
`get_username()` will at most cases return our OS's user account name. We can just use that. I'm not sure if we might ever have a failure here. Also since this is just for a local network, should we want to pass in the `owner` via CLI? It is just for testing right?

Reply:
> I'm not sure if we might ever have a failure here

If we do have a failure, we can use a default value like "owner" since it is a local network.

Reply:
as discussed, this can be addressed in follow up PR